src/libs/pr.ts

```diff
@@ -55,21 +64,24 @@ export async function handlePullRequestMessage(
 
   const summary = '<summary>Pulumi report</summary>';
 
-  const [htmlBody, trimmed]: [string, boolean] = ansiToHtml(output, MAX_CHARACTER_COMMENT);
+  const [htmlBody, trimmed]: [string, boolean] = ansiToHtml(output, MAX_CHARACTER_COMMENT, trimCommentsFromFront);
 
   const body = dedent`
     ${heading}
 
     <details>
     ${summary}
 
+    ${trimCommentsFromFront
```

Comment:
This is the reason why I added an extra new line. Previously, when the output was trimmed it would add a warning message to the end of the comment. My change makes it so in the event that we trim from the front it will add the warning message near the beginning of the comment, right before the trimmed output.

The blank line above was not there during the first pass I made and the tests passed without the addition of the extra `\n` to the tests. However when I used it in a project the markdown doesn't format properly. Instead of:

:warning: **Warn**: The output was too long and trimmed from the front.

It did:

:warning: \*\*Warn\*\*: The output was too long and trimmed from the front.